toolchain/check/check.cpp

```diff
@@ -227,11 +266,23 @@ static auto TrackImport(
     return;
   }
 
+  // Get the package imports. Avoid creating unused values for insert conflicts
+  // because there's a SmallVector.
+  auto package_imports_it =
+      unit_info.package_imports_map.find(import.package_id);
+  if (package_imports_it == unit_info.package_imports_map.end()) {
+    package_imports_it = unit_info.package_imports_map
+                             .insert({import.package_id, {.node = import.node}})
+                             .first;
+  }
```

Comment:
Hm, the designated initialization here makes this tricky. Maybe you could give `PackageImports` a constructor taking a `Parse::Node` and then use `try_emplace`:

```suggestion
  // Get the package imports.
  auto package_imports_it =
      unit_info.package_imports_map.try_emplace(import.package_id,
                                                import.node).first;
```

Non-blocking, though: I think this is OK as is.

Reply:
Done, I think this is a bit better.